README.md

```diff
@@ -72,7 +72,7 @@ Use the ``Deploy to Bluemix`` button **OR** create the services and run locally.
 > NOTE: These steps are only needed when running locally instead of using the ``Deploy to Bluemix`` button.
 
 1. [Clone the repo](#1-clone-the-repo)
-2. [Create Bluemix services](#2-create-bluemix-services)
+2. [Create Bluemix services](#2-create-watson-services-with-ibm-bluemix)
```

Comment:
The heading for different and link was broken https://github.com/IBM/watson-discovery-news/pull/21/files#diff-04c6e90faac2675aa89e2176d2eec7d8R87